test/specs/collections/Form/Form-test.js

```diff
@@ -37,6 +37,7 @@ describe('Form', () => {
   common.propKeyOnlyToClassName(Form, 'reply')
   common.propKeyOnlyToClassName(Form, 'success')
   common.propKeyOnlyToClassName(Form, 'warning')
+  common.propKeyOnlyToClassName(Form, 'inverted')
```

Comment:
Nit: alphabetical by prop name please.  I realize many are out of order, however, there is a current effort to update these.